src/string.cr

```diff
@@ -2940,7 +3073,25 @@ class String
     {pre, mid, post}
   end
 
-  def byte_index(byte : Int, offset = 0)
+  # Returns the index of *byte* in the string, or `nil` if the byte is not present.
```

Comment:
"Returns the index of the first ocurrence"?

Reply:
@straight-shoota what you think about this proposal? Is it not against previous considerations? I think my draft is in line with other docs in string file..

Reply:
I'm not sure, did we discuss this before? @RX14's suggestion sounds reasonable, it's more specific.

Reply:
Maybe, but it's same with current existing documentation for [index](https://crystal-lang.org/api/0.32.1/String.html#index(search:Regex,offset=0)-instance-method) method .. So maybe i have to change it too? And please write always complete formulation, what about for example:

1, _Returns byte index of the first ocurrence of *byte* in the string, or `nil` if not present._

or 

2, _Returns the index of the first ocurrence of *byte* in the string, or `nil` if not present._

or

3, _Returns the index of the first ocurrence in the string, or `nil` if not present._

or 

4, _Returns byte index of the first ocurrence in the string, or `nil` if not present._

or?

Reply:
Number 2 please.
And I'd like to change `#index` accordingly, that is add "first occurrence" to the existing phrase. For the regex overload: "first match"

Reply:
It seems this change was lost.